draft-ietf-privacypass-architecture.md

```diff
@@ -336,6 +356,12 @@ redeemed already in those redemption contexts, since these tokens can
 be issued and then spent multiple times in response to any such challenge.
 See {{Section 2.1.1 of AUTHSCHEME}} for discussion.
 
+How Clients respond to token challenges can have privacy implications.
+For example, if a token challenge admits a choice of Issuer, then the choice
```

Comment:
What does it mean for a "token challenge to admit a choice of Issuer"?

Reply:
As a hypothetical, imagine a token challenge specified more than one issuer, or if clients were given two challenges for two different issuers.

Reply:
I see. Our current challenge does not allow this to be expressed. How about:
```suggestion
For example, if an Origin allows the Client to choose an Issuer, then the choice
```